lib/fireblocks/api/transactions.rb

```diff
@@ -0,0 +1,44 @@
+module Fireblocks
+  class API
+    class Transactions
+      class << self
+        VALID_TRANSACTION_KEYS = [
+            :amount,
+            :assetId,
+            :source,
+            :destination,
+            :fee,
+            :gasPrice,
+            :note,
+            :autoStaking,
+            :networkStaking,
+            :cpuStaking
+        ]
+
+        def create(options)
+          body = options.slice(*VALID_TRANSACTION_KEYS)
+          Fireblocks::Request.post(body: body, path: '/v1/transactions')
+        end
+
+        def from_vault_to_external(amount:, asset_id:, source_id:, destination_id:, one_time_address:)
+          body = {
+            amount: amount,
+            assetId: 'ETH_TEST',
```

Comment:
@jfedgar Look good to me but was assetId supposed to be `assetId: asset_id` ?

Reply:
good catch, ha